core-plugins/docs/Joiner-batchjoiner.md

```diff
@@ -74,6 +74,23 @@ Setting it to true can cause a large drop in performance if there are a lot of n
 **Number of Partitions:** Number of partitions to use when grouping fields. If not specified, the execution
 framework will decide on the number to use.
 
+**Distribution:** Enabling distribution will increase the level of parallelism when joining skewed data. A 
+skewed join is when multiple records from one stage (the skewed stage) join to a single record in
+another stage. Distribution is possible when the following conditions are met:
+1. There are exactly two input stages. 
+1. Broadcast is not enabled for either stage.
```

Comment:
not sure why the numbers here are a different color. Are they part of the file or being added by github? Should make sure it doesn't render as a bunch of 1.s

Reply:
I've confirmed it looks correct in the UI, probably just Github highlighting 
![image](https://user-images.githubusercontent.com/54917202/88601619-f6ba6700-d025-11ea-8573-2303341fc50d.png)